rental.rb

```diff
@@ -0,0 +1,6 @@
+class Rental
+  def initialize(date)
+    @date = date
+  end
```

Comment:
- [ ] you're going to need to initialize the `book` and `person` as well in this method which will take both as arguments. and the rental should add itself to the instance variable created, which will look something like below:
```
    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
```